### DISCUSSION TITLE
get template and sequence V and J nucleotides

### DISCUSSION THREAD

**Discussion by gabrielathenes** [milaboratory/mixcr#1445] [Q&A]
Hello, is there a way to have access to this information with the exportAlign command line arguments ? Thanks a lot !
v_sequence_alignment	TCTGACGACACGGCCGTATATTACTGT
j_sequence_alignment	TGGGGCCGGGGGACC
v_germline_alignment	TCTGACGACACGGCCGTGTATTACTGT
j_germline_alignment	TGGGGCCAGGGCACC

**Comment by mizraelson** [comment]
Hi,
You mean you want columns with the sequences from the actual read and a germline sequence of the gene?

**Reply by gabrielathenes** [reply]
Yes, a column with the sequence from the v gene of the read and one from the v template

**Reply by mizraelson** [reply]
What part of the V gene do your reads cover? An entire sequence from FR1 to FR3?
